### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by CunliangGeng**
to solve issue #34 
Note that the package versions are automatically fetched from pypi. There are no version specifications in the code.

### CODE REVIEW

**Comment by jspaaks** [comment]
I tried (on Linux Ubuntu):
```
python3 -m venv venv3
source venv3/bin/activate
pip install -r requirements.txt
```
(but then I get complaints about ``openbabel==3.0.0``)

- If I comment out ``openbabel`` in ``requirements.txt``, I get complaints about ``rdkit``
- If I replace ``rdkit==2009.Q1-1`` with just ``rdkit``, it says there are no versions.
- If I install ``rdkit`` with ``pip install git+https://github.com/rdkit/rdkit@Release_2020_03_1b1``, I get an error about missing setup.py
- If I install ``rdkit`` with ``pip install "git+https://github.com/rdkit/rdkit@2dc66b2fd4e68a6af9afaa1e88c299b7bd60b8f4"``, still fails

Alternative could be to install from here https://blends.debian.org/debichem/tasks/cheminformatics

Come to think of it, rdkit==2009 seems really old, is this a typo?

**Comment by jspaaks** [comment]
```
sudo apt-get install python-rdkit librdkit1 rdkit-data
```
found it here https://github.com/rdkit/rdkit/blob/master/Docs/Book/Install.md

**Comment by jspaaks** [comment]
```
sudo apt install openbabel libopenbabel-dev python-openbabel
```

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 openbabel==3.0.0
-pandas==0.24.0
-PubChemPy==1.0.4
-pyteomics==4.1.2
-scipy==1.2.0
+pandas==0.24.2
+pubchempy==1.0.4
+pyteomics==4.2
+rdkit==2009.Q1-1
+scikit_learn==0.22.2.post1
```
Comment (jspaaks) [inline requirements.txt]: it seems to work both ways but I expected a dash instead of an underscore

**Review by jspaaks — APPROVED** [review]
I can't test it but based on a ``grep -nr 'import ' matchms/`` of the source code, this list seems much more complete than what used to be there.

**Comment by CunliangGeng** [comment]
> I tried (on Linux Ubuntu):
> 
> ```
> python3 -m venv venv3
> source venv3/bin/activate
> pip install -r requirements.txt
> ```
> 
> (but then I get complaints about `openbabel==3.0.0`)
> 
> * If I comment out `openbabel` in `requirements.txt`, I get complaints about `rdkit`
> * If I replace `rdkit==2009.Q1-1` with just `rdkit`, it says there are no versions.
> * If I install `rdkit` with `pip install git+https://github.com/rdkit/rdkit@Release_2020_03_1b1`, I get an error about missing setup.py
> * If I install `rdkit` with `pip install "git+https://github.com/rdkit/rdkit@2dc66b2fd4e68a6af9afaa1e88c299b7bd60b8f4"`, still fails
> 
> Alternative could be to install from here https://blends.debian.org/debichem/tasks/cheminformatics
> 
> Come to think of it, rdkit==2009 seems really old, is this a typo?

It's not a typo, on pypi the latest one is 2009  https://pypi.org/project/rdkit/#history.
But on conda, the lastest one is 2019  https://anaconda.org/conda-forge/rdkit

The `requirement.txt` is generated by `pipreqs` package, which search packages automatically on pypi.

**Comment by florian-huber** [comment]
Thanks Faruk and Christiaan for your reviews! I'll go ahead an merge it.

**Pull request merged**